.github/workflows/rosa-cost-report.yml

```diff
@@ -0,0 +1,31 @@
+name: ROSA/AWS - Cost Reporting
+
+on:
+  schedule:
+    - cron: '0 12 * * 7' # Runs At 12:00 PM UTC on Every Sunday
+  workflow_dispatch:
+
+jobs:
+  report:
+    name: Report on ROSA and AWS Costs for the current YTD, MTD and WTD
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Report on the Week-to-Date costs
+        run: bash -c "aws ce get-cost-and-usage --time-period Start=$(date -d "7 days ago" +"%Y-%m-%d"),End=$(date +%Y-%m-%d) --granularity=DAILY --metrics BlendedCost --output yaml"
```

Comment:
Would this work even without the `bash -c` prefix, or changed to have `shell: bash`?

Reply:
should be possible run without bash -c prefix., if thats our preference, just making sure to use bash shell in this case

Reply:
and it works without the `bash -c` just fine, will remove that in the final version.